test/test_logger_mixin.py

```diff
@@ -1,8 +1,9 @@
+import asyncio
 import unittest
 from logging import Logger, LogRecord
 from test.logger_mxin import LogLevel, TestLoggerMixin
 
-from hummingbot.logger import HummingbotLogger
+from hummingbot.logger.logger import HummingbotLogger
```

Comment:
why you are adding an extra .logger

Reply:
HummingbotLogger is defined in `hummingbot/logger/logger.py`. Otherwise it is underlined red in Pycharm